cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-common/management-credential-step/ManagementCredentials.tsx

```diff
@@ -217,21 +178,12 @@ function ManagementCredentials(props: Partial<StepProps>) {
         }
     };
 
-    const handleRefresh = async (event: MouseEvent<HTMLAnchorElement>) => {
+    const handleRefresh = (event: MouseEvent<HTMLAnchorElement>) => {
         event.preventDefault();
-        if (connectionStatus === CONNECTION_STATUS.CONNECTED) {
-            const initEC2KeyPairs = async () => {
-                setKeyPairLoading(true);
-                await AwsOrchestrator.initEC2KeyPairs({
-                    awsState,
-                    awsDispatch,
-                    errorObject,
-                    setErrorObject,
-                });
-                setKeyPairLoading(false);
-            };
-            initEC2KeyPairs();
-        }
+        setConnectionStatus(CONNECTION_STATUS.DISCONNECTED);
```

Comment:
From line 183 to line 186, only the connectionStatus is changed. I'm wondering why we are doing this in the handleRefresh?

Reply:
For `EC2KeyPair`, create another state value `refresh`. 

`handleRefresh` (line 182 in [ManagementCredentials.tsx](https://github.com/vmware-tanzu/community-edition/pull/5345/files#diff-58d78e216484096773033ad33ae612587870d1d997e75147e6479c2acb820c5b)) will change the `refresh` when clicked.

`useInitEC2KeyPairs` (line 71 in [AwsOrchestrator.service.tsx](https://github.com/vmware-tanzu/community-edition/pull/5345/files#diff-061adea28406c54b7c919fe9398e10908696203c867efbb039b8d75faa26f92d) ) will be executed when `refresh` value changed.

So EC2KeyPair will be initialized again.